config/routes.rb

```diff
@@ -1,4 +1,5 @@
 Rails.application.routes.draw do
+  devise_for :users
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
 
   # Defines the root path route ("/")
```

Comment:
### `README.md` file
![Screenshot from 2023-08-18 03-47-26](https://github.com/LarryIVC/larrys-blog/assets/83042742/b31398bd-7191-4539-bc0e-3deedb5b80e8)
- [ ] to make you r documentation looks more professional and descriptive, I think it would be better if you update your `Getting started` section with all information and correct ones needed by users in order to use your project locally, such as  `Prerequisites Tools`, how to `access` the cloned directory, `install` bundles, `create db`, `run/use` your app, `run tests`,... To help you, I suggest you refer this **[gist_file](https://gist.github.com/tresorsawasawa/4cde69cff1b40161b9de380f9891f415)**

Reply:
Hi @tresorsawasawa, thanks for your feedback, it's done. ✔️